gradle/test.gradle

```diff
@@ -225,6 +225,11 @@ subprojects {
 
         maxHeapSize '768m'
         jvmArgs = ['-XX:+HeapDumpOnOutOfMemoryError', '-ea']
+        if ( testJVMVer.toInteger() >= 9 ) {
+          jvmArgs += ["--add-opens", "java.xml/jdk.xml.internal=ALL-UNNAMED"]
```

Comment:
Just curious, is this related to adding the JAXB dependencies or does it solve a different issue?

Reply:
this one is related to LogService, I believe